debian/postinst

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+# postinst script for pac4cli
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+install_virtualenv() {
+    echo "Stopping pac4cli service (if it is running)..."
+    systemctl stop pac4cli.service || true
+    virtualenv -p python3 --system-site-packages /opt/pac4cli
+    /opt/pac4cli/bin/pip install txdbus systemd
```

Comment:
Since we opted to install the rest of dependencies system-wide, why do we install those specifically in a virtualenv?